app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -140,6 +140,13 @@ static void addEventMetadata(Event event) {
         event.setAppInstallId(Prefs.getAppInstallId());
     }
 
+    /**
+     * Send all scheduled-and-waiting events in case of app being suspended.
```

Comment:
This comment might be too specific: this function might be called for other reasons than the app being suspended.  I'm not sure this needs a comment at all, since `flushCachedEvents()` is a perfectly self-documenting name.

Reply:
Correct... makes sense... removed the comment.